wagtail/admin/static_src/wagtailadmin/scss/layouts/login.scss

```diff
@@ -114,16 +118,25 @@ input[type=checkbox]:before {
     line-height: 1.6em;
 }
 
+// Forgotten pasword link
 .help {
-    opacity: 1;
-    position: absolute;
     font-size: 1.3em;
-    margin-top: -0.5em;
-    right: 5%;
-    top: 213px;
+    margin-top: 0;
+    padding: 10px 0;
 
     @include media-breakpoint-up(sm) {
-        top: 65%;
+        position: absolute;
+        top: 139px;
+        right: 5%;
+        padding: 0;
+    }
+
+    &__link {
+        color: $color-white;
```

Comment:
Setting the link to be white at mobile as teal on black is not very visible - but I don't know if there is already a standard link style in this case.

Reply:
Not that I know of. This looks great!